framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -399,13 +399,12 @@ protected final AnnotationMirrorSet getAnnotationsField() {
     // getAnnotations in javac APIs works.
     // Removed getEffectiveAnnotation
     public AnnotationMirrorSet getEffectiveAnnotations() {
-        AnnotationMirrorSet effectiveAnnotations = getErased().getAnnotations();
```

Comment:
In several places this removes local variables and directly returns an expression.
The local variables were useful when adding debugging statements before the return.
This is particularly visible here, where now the following comments are nonsense, as there no longer is an `effectiveAnnotations` variable.
I think I only saw one time where I would remove the local variable - I'll comment there.